apps/mull-ui/src/app/pages/create-event/date-calendar/date-calendar.tsx

```diff
@@ -52,6 +52,8 @@ const DateCalendar = ({
           onFocusChange={handleFocusChange}
           locale={enGB}
           minimumDate={new Date()}
+          // @ts-ignore
+          touchDragEnabled={false}
```

Comment:
💡 mute linting issue (aka tell eslint to shut up about us telling typescript to shut up)

```suggestion
          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
          // @ts-ignore
          touchDragEnabled={false}
```